docs/paper/admin/how-to/basic-troubleshooting.md

```diff
@@ -0,0 +1,143 @@
+The thread dump from Paper Watchdog can be found below that line. If you find any plugin's name in there, talk to the plugin's author about that, not PaperMC.
+
+### Case 02: Stack Trace
+
+Almost every problem you encounter will print error message lines, which are called "stack trace", on the server console. Examining the stack trace will help you find out what is causing problems on your server.
+
+The stack trace starts with the error message, exception type, and exception message.
+
+Both error messages and exception messages were put there by the developer of either your plugin or paper server. These messages tell you what problem your server experienced.  
+An exception type like `java.lang.RuntimeException` tells you the type of the exception. This will help the developer (and you) understand the type of problem.
+
+Many lines beginning with `at` may appear beneath the exception message. These are the body of the stack trace. These lines tell you where the problem starts. The top line of the body of the stack trace will tell you exactly where the problem occurred and, if possible, display where it came from.
+
+If you find any plugin's name in the stack trace, head to [Check Plugin Updates](#check-plugin-updates) and read from there. In most cases, the plugin, whose name is found on the stack trace, is causing the problem. If not, continue reading.
+
+Here are some examples of stack trace.
+
+<details>
+  <summary>Example 01: Server attempted to load chunk saved with newer version of minecraft!</summary>
+
+```plaintext
+[00:00:00 WARN]: java.lang.RuntimeException: Server attempted to load chunk saved with newer version of minecraft! 3218 > 3120
+```
+
+You tried to load the world generated with a higher version of Minecraft. You cannot do this.  
+If you don't have any backup of your world before the chunk version update, you must use your updated world with a higher version of Minecraft.
+
+</details>
+
+<!-- Another examples -->
+
+## Find Out What Plugin Is Causing The Problem
+
+If you can't find the name of any plugin in the thread dump or stack trace, try these steps.
+
+### Disable All Plugins
+
+To determine if it is a plugin or the paper itself that is causing problems, disable all of your plugins first.
+
+You can disable all of your plugins by renaming the `plugins` directory to something else, such as `plugins-disabled`, or by archiving the `plugins` directory and deleting it.
+
+After that, try to run your server.
+
+If the problem is resolved after removing the plugins, you know that it was a plugin that caused the issue.  
+If you still experience problems, head to [Paper Documentation](#paper-documentation). Maybe your paper server is misconfigured, and that is creating issues.
+
+### Binary Search
+
+To find out what plugin is causing problem, [binary search](https://en.wikipedia.org/wiki/Binary_search_algorithm) is ideal way to do it.
+
+<!-- Animated GIF file -->
+
+And here is brief explanation about how to do it.
+
```

Comment:
```suggestion
To efficiently search for the plugin that is causing the problem, you can do the following:
```

Reply:
What about the title, `### Binary Search`? Do you think it is enough or should this also be changed according to the text below of it?